test/utils.sh

```diff
@@ -222,24 +222,26 @@ cli_get_pods_test_env () {
 
 test_secret_is_provided () {
   secret_value=$1
+  variable_name="${2:-secrets/test_secret}"
+  environment_variable_name="${3:-TEST_SECRET}"
 
   set_namespace "$CONJUR_NAMESPACE_NAME"
   conjur_cli_pod=$(get_conjur_cli_pod_name)
-  $cli_with_timeout "exec $conjur_cli_pod -- conjur variable values add secrets/test_secret $secret_value"
+  $cli_with_timeout "exec $conjur_cli_pod -- conjur variable values add \"$variable_name\" $secret_value"
```

Comment:
isn't `$variable_name` enough here since the value of variable_name is already wrapped in `""`

Reply:
and also matches with what you have below
`'$environment_variable_name' `

Reply:
> isn't `$variable_name` enough here since the value of variable_name is already wrapped in `""`

No because this text is then sent to a command that needs the quotes in case of spaces in the variable name.